app/models/vm_or_template.rb

```diff
@@ -1923,6 +1923,10 @@ def validate_supported
     {:available => true,   :message => nil}
   end
 
+  def validate_unsupported(message_prefix)
```

Comment:
@roliveri btw. the patch that added ssa check has forbid SSA on all MiqTemplate classes, is it on purpose?